### ISSUE TITLE
The parameters of ‘reporter’ in the doc

### ISSUE THREAD

**Issue by HollrayChan** [awslabs/autogluon#232]
Hello，I am using autogluon on my pytorch network, and I had read the doc(https://autogluon.mxnet.io/tutorials/torch/hpo.html).
But I have a question about the 'reporter' and I didn't find any detailed info about it in the doc.
In my code of test, it would return two lists ——mAP((mean average precision) and CMC(Cumulative Matching Characteristic).

def test():
...
    return result.mAP, list(result.CMC[0: 150]) 

Instead of  accuracy in the example, does reporter support more indicators or  custom evaluation indicators？
If autogluon support , the code of reporter as follow . 

reporter(epoch=epoch, mAP=mAP)

I didn't answer in https://autogluon.mxnet.io/tutorials, or can it be solved by other methods？ I hope get help, thanks~

**Comment by zhanghang1989** [comment]
Thanks for interest in Customizing AutoGluon!
`reporter` is used to communicate between the training code and `schedulers`.
In your example, you can use the following code:
```
reporter(epoch=epoch, mAP=mAP, CMC=cmc)
```
and set `time_attr='epoch'` and `reward_attr='mAP'`:
https://autogluon.mxnet.io/api/autogluon.scheduler.html?highlight=fifo#autogluon.scheduler.FIFOScheduler
